packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -55,34 +62,24 @@ $skeleton-color-end: rgba($gray1, 0.2) !default;
   }
 }
 
-// Some properties should not be overriden. A more specific explanation is given before each usage
-// of "!important".
+// This class hides content with a glowing, rounded rectangle. CSS properties that should not be overriden by consumers use the
```

Comment:
this line is too long for the linter, please rewrap to 100 characters per line.
merge latest master and run `gulp check` and you'll see what i mean.